tools/lighter/lighter.xml

```diff
@@ -32,7 +32,7 @@
     ]]>
     </command>
     <inputs>
-        <param argument="-r" multiple="True" type="data" label="Input file"/>
+        <param argument="-r" multiple="True" type="data" format="data" label="Input file"/>

```

Comment:
The file can be fasta and fastq, and can be gzip'ed with extension *.gz. 
                     When the input file is *.gz, the corresponding output file will also be gzip'ed.

From https://github.com/mourisl/Lighter

Reply:
Thanks:)